.travis.yml

```diff
@@ -49,7 +49,8 @@ jobs:
       script: npm run build-gh-release
       deploy:
         provider: releases
-        api_key: $GITHUB_TOKEN
+        edge: true
+        api_token: $GITHUB_TOKEN
```

Comment:
I think `api_token` needs to be just `token` https://docs.travis-ci.com/user/deployment-v2/providers/releases/#known-options for the `releases` provider

Reply:
Good spot, changed that to `token`